src/screens/StudyScreen/WordCard.jsx

```diff
@@ -29,17 +30,23 @@ const WordCard = ({ display, wordData, language, fontSize, fontStyle, onCopyText
       {wordData.detail.map(({ meaning, partsOfSpeech, example }) => (
         <Box key={`${partsOfSpeech}-${meaning[LANGS.en].slice(0, 20)}-${example.slice(0, 20)}`} pt={8}>
           <Stack space={3}>
-            <DisplayText size={getTextSize(fontSize)} fontStyle={fontStyle}>
-              {`${PARTS_OF_SPEECH_SHORTHAND[partsOfSpeech]} ${meaning[LANGS[language]] || meaning[LANGS.en]}`}
-            </DisplayText>
-            {showBilingual && (
-              <DisplayText size={getTextSize(fontSize)} fontStyle={fontStyle} colorLight='base.gray' colorDark='base.gray'>
-                {`${PARTS_OF_SPEECH_SHORTHAND[partsOfSpeech]} ${meaning[LANGS.en]}`}
+            <Pressable onLongPress={() => onCopyText(meaning[LANGS[language]] || meaning[LANGS.en])}>
+              <DisplayText size={getTextSize(fontSize)} fontStyle={fontStyle}>
+                {`${PARTS_OF_SPEECH_SHORTHAND[partsOfSpeech]} ${meaning[LANGS[language]] || meaning[LANGS.en]}`}
```

Comment:
There's a lot of duplicate code here, e.g. `meaning[LANGS[language]] || meaning[LANGS.en]}`. Extract and transform all the necessary values in advance, so the component can be more readable.